mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -317,6 +331,17 @@ def has_stack(self) -> bool:
         """
         return self._stack is not None
 
+    def has_normalise_stack(self) -> bool:
+        """
+        Check if data is available to export
+        @return: True if data is available to export and False otherwise
+        """
+        return self._normalise_stack is not None
+
+    def apply_shuttercount_correction(self) -> bool:
+        """Return True if there are no shuttercount issues and False otherwise"""
+        return self.shuttercount_issue() == ""
```

Comment:
This needs renaming, the current name makes it sound like it applies the correction.

Also not sure why it is used when exporting to a file. Shouldn't those use `self.view.shuttercount_norm_enabled()`

Reply:
Looked into this more can i can actually drop this now, made sense before i refactored a lot, but taking a step back, i can actually just use `self.view.shuttercount_norm_enabled()` like you suggest